src/hotspot/cpu/x86/matcher_x86.hpp

```diff
@@ -261,6 +261,27 @@
     }
   }
 
+  static bool is_vector_cast_supported(BasicType from_bt, BasicType to_bt) {
```

Comment:
Why does this not live next to `Matcher::match_rule_supported_vector`, would that not be a better fit?